Directory.Build.targets

```diff
@@ -21,4 +21,15 @@
     -->
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
+
+  <!-- Packaging -->
+  <PropertyGroup>
+    <PackageDescription Condition="'$(PackageDescription)' == '' and '$(Description)' != ''">$(Description)</PackageDescription>
+    <PackageDescription Condition="'@(PackageDescriptionCommonType)' != '' and '$(PackageDescription)' != ''">$(PackageDescription)%0A%0ACommonly Used Types:@(PackageDescriptionCommonType->'%0A%(Identity)')</PackageDescription>
```

Comment:
i am little bit unclear on where this will executed. We are adding concatenating the common types in the target ?

Reply:
You are looking at an old commit :)